examples/scripts/making_a_bot.py

```diff
@@ -50,10 +51,11 @@ def run(self) -> None:  # overwrite the AbstractBotJob run method
                 self.logger.info(
                     '\t'.join([olid, old_title, edition.title])
                 )  # don't forget to log modifications!
```

Comment:
Please move the comment to the line above so the rest of the code fits on a single line.